CONTRIBUTING.md

```diff
@@ -351,6 +351,226 @@ The third schema is `third.schema.json`, it extends both `second`, and transitiv
+
+The following relationship types are defined by XDM:
+
+* `xdm:descriptorOneToOne`: describes a 1:1 relationship between a source schema and a destination schema
+* `xdm:descriptorOneToMany`: describes a 1:m relationship between a source schema and a destination schema
+* `xdm:descriptorManyToMany`: describes an m:n relationship between a source schema and a destination schema
+
+These relationships are defined in XDM using the RelationshipDescriptor schema.
+
+#### Update Policies
+
+Data described by an XDM schema may change over time, and as such a data object may reflect an update of a previous instance of that object. There are different ways that an update may be handled, and this way depends both on the nature of the data and the specific application it is being used for.
+
+XDM defines a schema descriptor of type `xdm:descriptorUpdatePolicy`, which describes several common methods of handling an update:
+
+* `xdm:updateMerge`: the data in the new object should be merged into the existing object; the method by which a merge is applied is defined by the application
+* `xdm:updateReplace`: the new data object should replace the existing data object
+* `xdm:updateTimeSeries`: the data is time series data, and the new object should be logged/collected without changing any existing data
+
+Update policies are defined using the UpdatePolicyDescriptor schema.
+
+#### Other Supported Schema Descriptors
+
+A number of additional schema descriptors are defined by XDM:
+
+* `xdm:descriptorIdentity`: allows a property in a schema to be used as an identity, even if it does not conform to the Identity schema.
+* `xdm:descriptorPrimaryKey`: allows a property other than `@id` to be flagged as the primary key for a schema
+* `xdm:descriptorInstantiable`: allows a schema to be flagged as 'instantiable', which may be used to differentiate schemas that define primary business objects versus supporting schemas intended to be embedded in another schema.
+
+#### Schema Descriptor Examples
+
+##### Example Relationship Descriptor
+
+We have two schemas, which form a parent/child relationship. The first is parent.json:
+
+```json
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/xdm/example/parent",
+  "title": "Parent",
+  "type": "object",
+  "meta:descriptors": {
+    "@type": "xdm:descriptorOneToMany",
+    "xdm:sourcePropery": "@id",
+    "xdm:destSchema": "https://ns.adobe.com/xdm/example/child",
+    "xdm:destProperty": "xdm:parent"
+  },
+  "properties": {
+    "@id": { "type": "string" }
+  }
+}
+```
+
+The second is child.json:
+
+```json
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/xdm/example/child",
+  "title": "Child",
+  "type": "object",
+  "properties": {
+    "@id": { "type": "string" },
+    "xdm:parent": { "type": "string" }
```

Comment:
I would include `format: uri`, as the `xdm:parent` property can have only values that would be valid `@id`s in `parent.schema.json`